src/markdown_generators_test.js

```diff
@@ -478,9 +478,9 @@ describe("turndown service", () => {
   })
 
   it("should generate an anchor shortcode for an a tag with a name attribute", () => {
-    const inputHTML = `<a name="test"></a>`
+    const inputHTML = `<a name="test">test</a>`
     const markdown = markdownGenerators.turndownService.turndown(inputHTML)
-    assert.equal(markdown, `{{< anchor "test" >}}`)
+    assert.equal(markdown, `{{< anchor "test" >}}test{{< /anchor >}}`)
```

Comment:
kind of silly - but maybe good to just have a quick test that it pulls out an `href` too?

Reply:
done